### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites_Screen Reader: User is navigating to different page on pressing Enter on "Clickable" after button "More"

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#182]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Google Chrome Version 68.0.3440.106 
Screen Reader: NVDA
MAS ID Violated: MAS1.3.2,MAS1.3.1

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till "Local IIS" in the top navigation pane and Press Enter.
5) Tab till "Local IIS" link under Servers and Press Enter.
7) Tab till "More" button and press Tab.
8) Verify Screen Reader is reading an extra content "Clickable" and on pressing Enter on that user is navigating to different page or not

**Actual Result:**
User is navigating to different page on pressing Enter on "Clickable" after button "More"

**Expected Result:**
NVDA should not read any extra content(Clickable) which is not visible on screen and also user should not navigate to other page on pressing Enter on Clickable 

**User Impact:** 
Users who are blind or have impaired vision will face difficulty in navigating if screen reader reads any extra content that is not present on the screen and also navigating to other pages can create inconvenience to the user

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachments:**
[Bug_182_On pressing Enter on clickable focus is going to the different page.pptx](https://github.com/Microsoft/IIS.WebManager/files/2335400/Bug_182_On.pressing.Enter.on.clickable.focus.is.going.to.the.different.page.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Chrome; #NVDA; #MAS1.3.2

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
